<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ADMIN" failures="2" tests="16" name="Default test(failed)(failed)" time="0.405" errors="0" timestamp="2022-08-03T13:16:36 IST">
  <testcase classname="testng_Failed.FirstClass" name="testMethod1" time="0.051"/>
  <testcase classname="testng_Failed.FirstClass" name="testMethod2" time="0.005"/>
  <testcase classname="testng_Failed.FirstClass" name="testMethod3" time="0.014"/>
  <testcase classname="testng_Failed.FirstClass" name="testMethod4" time="0.003"/>
  <testcase classname="testng_Failed.FirstClass" name="testMethod5" time="0.011">
    <skipped/>
  </testcase> <!-- testMethod5 -->
  <testcase classname="testng_Failed.FirstClass" name="testMethod5" time="0.003">
    <skipped/>
  </testcase> <!-- testMethod5 -->
  <testcase classname="testng_Failed.FirstClass" name="testMethod5" time="0.008">
    <skipped/>
  </testcase> <!-- testMethod5 -->
  <testcase classname="testng_Failed.FirstClass" name="testMethod5" time="0.002">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testng_Failed.FirstClass.testMethod5(FirstClass.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testMethod5 -->
  <testcase classname="testng_Failed.Second_Class" name="testMethod1" time="0.002"/>
  <testcase classname="testng_Failed.Second_Class" name="testMethod2" time="0.016"/>
  <testcase classname="testng_Failed.Second_Class" name="testMethod3" time="0.003"/>
  <testcase classname="testng_Failed.Second_Class" name="testMethod4" time="0.005"/>
  <testcase classname="testng_Failed.Second_Class" name="testMethod5" time="0.004">
    <skipped/>
  </testcase> <!-- testMethod5 -->
  <testcase classname="testng_Failed.Second_Class" name="testMethod5" time="0.013">
    <skipped/>
  </testcase> <!-- testMethod5 -->
  <testcase classname="testng_Failed.Second_Class" name="testMethod5" time="0.014">
    <skipped/>
  </testcase> <!-- testMethod5 -->
  <testcase classname="testng_Failed.Second_Class" name="testMethod5" time="0.007">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testng_Failed.Second_Class.testMethod5(Second_Class.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testMethod5 -->
</testsuite> <!-- Default test(failed)(failed) -->
